apps/internal/urls.py

```
@@ -0,0 +1,9 @@
+import itertools
```

Ruff

`itertools` imported but unused

Remove unused import: `itertools`

(F401)